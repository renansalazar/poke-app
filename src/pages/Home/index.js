import Buscador from '../../components/Buscador';
import Lista from '../../components/Lista';


export default function Home () {
  return (
    <>
      <section>
        <Buscador />
      </section>
      <section className>
        <Lista />
      </section>
    </>
  )
}