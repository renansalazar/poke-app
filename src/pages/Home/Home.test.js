import React from 'react';
import {render} from '@testing-library/react';
import  Home from './';
import { QueryClient, QueryClientProvider } from 'react-query';

describe('Detalle component', () => {
  test('first call to detail', () => {
    const queryClient = new QueryClient();
    const { container } = render(
      <QueryClientProvider client={queryClient}>
        <Home />
      </QueryClientProvider>)
    
    expect(container.firstChild).toMatchSnapshot()
  });
});