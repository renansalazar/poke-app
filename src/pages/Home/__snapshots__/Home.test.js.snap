// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Detalle component first call to detail 1`] = `
<section>
  <form
    class="containerBuscador"
  >
    <input
      data-testid="buscadorText"
      placeholder="Inserte Nombre de Pokemon"
      type="search"
      value=""
    />
    <button>
      Buscar
    </button>
  </form>
</section>
`;
